cmd/ocm/create/cluster/cmd.go

```diff
@@ -75,6 +80,12 @@ func init() {
 
 	arguments.AddProviderFlag(fs, &args.provider)
 	arguments.AddCCSFlags(fs, &args.ccs)
+
+	fs.Var(
+		&args.gcpServiceAccount,
+		"service-account-file",
```

Comment:
WDYT about renaming this `--gcp-service-account-file`?  4 more characters to type but maybe worth it to keep the flags organized.

(BTW, maybe it's time to start using Cobra's flag groups to organize the help :thinking:)